### PR TITLE
Node/Acct: Renaming

### PR DESCRIPTION
**Pull request by bruce-riley**
Change "Accounting" to "Accountant".

Change-Id: I20783b4ae4e445a54d45280e19332af87336a92a

### CODE REVIEW

**Comment by jynnantonix** [comment]
I tried changing the base branch of the PR to #2181 but it looks like one of them has been rebased on main while the other one has not.  Since this PR includes *all* the changes from #2181 it's a bit hard to see exactly what's different.

I suggest using the graphite cli to stack this PR on top of #2181.  Otherwise we can wait until after #2181 is merged and then rebase this PR before reviewing.

**Review by hendrikhofstadt — CHANGES_REQUESTED** [review]
pls rebase

**Review by hendrikhofstadt — APPROVED** [review]

**Review by jynnantonix — APPROVED** [review]

**Pull request merged**
